code/modules/research/designs/machine_designs.dm

```diff
@@ -485,6 +511,7 @@
 	name = "Machine Design (NTNet Relay Board)"
 	desc = "The circuit board for a wireless network relay."
 	id = "ntnet_relay"
+	build_type = IMPRINTER
```

Comment:
Undoced balance change.

Reply:
This could only be printed from the IMPRINTER before. Same as above. 

![image](https://user-images.githubusercontent.com/24975989/109794878-3a093380-7c0e-11eb-90e7-2636fc9adc3b.png)
![image](https://user-images.githubusercontent.com/24975989/109795435-d29fb380-7c0e-11eb-8933-3cdc1370a66f.png)

Although NTnet is just used for door remotes and the mulebot (?) and not teleportation as I originally thought.

There's also nothing to be gained by including more chaff that no ghost roles can actually use. I did the same when I trimmed the cargo consoles out. Not because they're something ghost roles shouldn't have, but because they're things ghost roles can't use. I'll move it to the appropriate category in the PR body. Unless you're telling me there's some massive use-case for ghost roles to build the quantum relay?